.travis.yml

```diff
@@ -1,9 +1,12 @@
+dist: trusty
```

Comment:
It appears this distribution applies to all versions of Ruby being tested? Trusty reached End Of Life on April 30th, 2019. I don't think it makes sense to continue testing on this.  If the justification is to support older Ruby versions, then maybe we need to drop support for those instead.  I don't see a lot of value in continuing to support old versions of Ruby going forward. Anyone on an older Ruby can use a similarly older version of Ruby Speech.

Reply:
@bklang That's a good point.  I know that we tout Rubinius compatibility, but I haven't seen it be practical to use Rubinius with a recent version of a Travis image, unfortunately.

Using `dist: trusty` got us a little further in the right direction (the only image that successfully rvm-installs "rbx-3" and "rbx-4"), but it would still choke on trying to natively compile extensions for `ffi`.  Since trusty still can't effectively get us to a place where it actually runs the builds for rbx, then it doesn't provide any value.

So yeah, we should probably use something more recent and leave rbx in "allow_failures" for the time being.

Reply:
We always wanted to support Rubinius, but I've never heard of anyone actually using it. Even in CI, Rubinius has always been marked to `allow_failures`.  I definitely would not consider it a platform with full support. It's a nice-to-have.